docs/installingNative.md

```diff
@@ -181,48 +194,21 @@ Most of these prerequisites can be installed using the package manager of your O
 
   You may need to restart the PostgreSQL server after changing the file above.
 
-- Run the test suite (Docker must be installed and running):
-
-  ```sh
-  make test
-  ```
-
-- Run the linters:
-
-  ```sh
-  make lint # or lint-js for Javascript only, or lint-python for Python only
-  ```
-
-- Create the file `PrairieLearn/config.json` with the path of your local course repository and with the path of a directory into which temporary files will be saved when using the in-browser file editor (edit both paths as needed):
-
-  ```json title="config.json"
-  {
-    "courseDirs": ["/Users/mwest/git/pl-tam212", "exampleCourse"],
-    "filesRoot": "../filesRoot",
-    "workspaceHostHomeDirRoot": "/tmp/workspace",
-    "workspaceHomeDirRoot": "/tmp/workspace"
-  }
-  ```
-
-- Run the server in development mode to automatically restart when changes are detected:
-
-  ```sh
-  make dev
-  ```
-
-  Alternatively, you can build and run the code to more closely mimic what will happen in production environments:
+## Configuration
 
-  ```sh
-  make build
-  make start
-  ```
+If you have your own [PrairieLearn course repository](./requestCourse/index.md), you will need to create the file `PrairieLearn/config.json` with the path of your local course repository. If you need support for workspaces, you should provide a path to a directory into which temporary files will be saved when using the in-browser file editor. Here is a sample configuration:
```

Comment:
As I mentioned in a previous comment, "you should provide a path to a directory into which temporary files will be saved when using the in-browser file editor" is a non sequitur from "if you need support for workspaces". `filesRoot` is not related to workspace support.

Reply:
It is unclear to me how to explain `workspaceHostHomeDirRoot`, perhaps you have some  insight there?

Reply:
Maybe you're getting confused about what "in-browser file editor" is referring to? That's not related to workspaces, though I could see how one might think it is. It's referring to the in-browser file editor for course content itself, like this:

<img width="931" alt="Screenshot 2025-05-12 at 15 47 33" src="https://github.com/user-attachments/assets/1eb010aa-44f0-4610-aaa6-e2c01e917e2c" />

Supporting that is why `filesRoot` must be set.

Reply:
To clarify -- I had updated the text to:

> If you need support for the in-browser file editor or file uploads, you should set `filesRoot`. If you need support for workspaces, you should provide a path to a directory into which temporary files will be saved. Here is a sample configuration:

I didn't have a great explanation for what exactly `workspaceHostHomeDirRoot` and `workspaceHomeDirRoot` do.

Reply:
I didn't see that update, there wasn't a commit linked above so I was just responding to the lines that GitHub showed for this comment thread. Your changes look good.